app/v1/bank/keeper.go

```diff
@@ -18,6 +19,15 @@ const (
 	costAddCoins       sdk.Gas = 10
 )
 
+var GovDepositCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("govDepositedCoins")))
+var ServiceDepositCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("serviceDepositedCoins")))
+var ServiceRequestCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("serviceRequestCoins")))
+
+var CommunityTaxCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("communityTaxCoins")))
+var ServiceTaxCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("serviceTaxCoins")))
+
+var BurnedCoinsAccAddr = sdk.AccAddress(crypto.AddressHash([]byte("burnedCoins")))
```

Comment:
I suggest to use 0x00 as burned address. Otherwise, you have to convince users that this is a dead address.

Reply:
> I suggest to use 0x00 as burned address. Otherwise, you have to convince users that this is a dead address.

Good idea.